spdx-utils/src/main/resources/declared-license-mapping.yml

```diff
@@ -371,6 +371,7 @@
 "The W3C License": W3C
 "The W3C Software License": W3C
 "Three-clause BSD-style": BSD-3-Clause
+"Trilead Library License (BSD-Like)": BSD-3-Clause
```

Comment:
See bottom of https://svnkit.com/TRILEAD-LICENSE.txt it not just BSD-3-Clause but BSD-3-Clause AND MIT

Reply:
Ah, thanks. I overlooked that the code from Bouncy Castle was actually copied and modified. Fixed.